app/scripts/modules/core/instance/instances.directive.js

```diff
@@ -7,15 +7,15 @@ let angular = require('angular');
 module.exports = angular.module('spinnaker.core.instance.instances.directive', [
   require('../utils/jQuery.js'),
 ])
-  .directive('instances', function ($timeout, $) {
+  .directive('instances', function ($rootScope, $timeout, $) {
     return {
       restrict: 'E',
       scope: {
         instances: '=',
         highlight: '=',
       },
       link: function (scope, elem) {
-        var $state = scope.$parent.$state;
+        var $state = $rootScope.$state;
```

Comment:
It's been a long time since I wrote this, but I recall picking up the `$state` from the parent element to make sure we could build relative links - worth verifying you can click on an instance, then another instance, as well as going from a server group or load balancer and back to the instance.